index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+</head>
+
+<body>
+  <header id="brand" class="toolbar">
+    <div class="my-logo">
+      Richard Burneo
+    </div>
+    <nav class="menu-botton">
+      <i class="fa-solid fa-bars"></i>
+    </nav>
+  </header>
+  <section class="headline">
+    <div class="information">
+      <h1 class="my-name">Hey there. <br>I'm Richard</h1>
+      <h2 class="iam">I'm Sotfware Developer</h2>
+      <p class="my-description">I can help you build a product , feature or website Look through some of my work and
+        experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.</p>
+    </div>
+    <div class="social-network">
+      <ul>
+        <li>
+          <i class="fa-brands fa-github"></i>
+        </li>
+        <li>
+          <i class="fab fa-linkedin-in"></i>
+        </li>
+        <li>
+          <i class="fa-brands fa-angellist"></i>
+        </li>
+        <li>
+          <i class="fa-brands fa-twitter"></i>
+        </li>
+        <li>
+          <i class="fa-solid fa-m"></i>
+        </li>
+      </ul>
+    </div>
+  </section>
```

Comment:
[Optiona Changes ]  Kindly style your social media icons in this section to be on the left. If you take a look at the design, the icons are placed on the left. Please fix that as

[Optiona Changes ]  Also please add some spacing from the top of the header to the heading text. The previous reviewer did not spot that, but it's required your style matches the design in the Figma template. Please fix that 👍 

**Your Design**

![image](https://user-images.githubusercontent.com/50941074/164933669-25df0ef3-451a-436e-b6ac-02fe3b37dc1d.png)

**The Figma Design**
![image](https://user-images.githubusercontent.com/50941074/164933136-0cff44c7-4685-45d2-902a-6a6ed0109c28.png)